packages/site/pages/components/screenreaderonly.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import {
+  Chrome,
+  Code,
+  Content,
+  P,
+  PageHeading,
+  SectionHeading
+} from '../../src/ui/index.js'
+
+export default () => (
+  <Chrome>
+    <Content title="ScreenReaderOnly">
+      <PageHeading packageName="screenreaderonly">ScreenReaderOnly</PageHeading>
+
+      <P>Install the component dependency:</P>
+      <Code language="bash">
+        npm install @pluralsight/ps-design-system-screenreaderonly
+      </Code>
+
+      <P>Include a React component in your project:</P>
+      <Code language="javascript">
+        import Typeahead from '@pluralsight/ps-design-system-screenreaderonly'
+      </Code>
+
+      <SectionHeading>Usage example</SectionHeading>
```

Comment:
In a future version of docs should we include more detailed explanation of why and how to use this?

Reply:
yes. i was being lazy 😬 i'll update